### ISSUE TITLE
508-defect-3 [COGNITION] Eligibility page, links should be styled as links

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22840]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

## Details

The "Go to eBenefits" link within the eligibility page is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click go to xyz _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link, and even more so when navigating by links/buttons using the rotor
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

## Acceptance Criteria

- [ ] Links are styled as links

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution 

Use the new action link component in the VA design system

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [WCAG Success Criterion 3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Screenshots or Trace Logs

<img width="1020" alt="Screen Shot 2021-04-07 at 11 44 42 AM" src="https://user-images.githubusercontent.com/14154792/113895789-4c3f3a00-9797-11eb-8227-8e7c91645ba8.png">

**Comment by micahchiang** [comment]
@jason-gcio @joshkimux - This seems to be another issue that needs to be fixed in Drupal. It's weird because I think this page (I assume it's `/careers-employment/vocational-rehabilitation/how-to-apply`) was originally using our [CTA widget](https://github.com/department-of-veterans-affairs/vets-website/blob/f482361eda1e6dea3a5928472e1eed1f20355531/src/applications/static-pages/vre-chapter31/Chapter31CTA.js#L6-L11) which does use an action link, but at some point it must have been replaced by a hard coded link in drupal.

**Comment by jason-gcio** [comment]
Updated screenshot at page in question: https://www.va.gov/careers-employment/vocational-rehabilitation/eligibility/
![image](https://user-images.githubusercontent.com/53826255/135471727-263a7d1f-c82b-4a58-bf9b-75e847d64a4f.png)

**Comment by jason-gcio** [comment]
Request: https://dsva.slack.com/archives/C52CL1PKQ/p1633011698033000

**Comment by jason-gcio** [comment]
This ticket is blocked by this ticket: department-of-veterans-affairs/va.gov-cms#6604

**Comment by ohgreatkate** [comment]
resolved.
